schemas/dbt/manifest/v12.json

```diff
@@ -9121,19 +9133,7 @@
                         "type": "integer"
                       },
                       "granularity": {
-                        "enum": [
```

Comment:
Not sure why this gets changed

Reply:
I checked out main branch and generated a schema, these changes are also there, so it is not particular to this PR